bin/colors

```diff
@@ -13,22 +13,22 @@ extension Dictionary {
 let inPath = "Design/Colors.json"
 let outPath = "Library/Styles/Colors.swift"
 
-let data = NSData(contentsOfFile: inPath)!
-data.length
+let data = try! Data(contentsOf: URL(fileURLWithPath: inPath))
```

Comment:
yahhh mostly cause for a script it would be better to crash to let us know something went wrong than to silently fail. i could also do a `try?` and then output a message saying something went wrong if we got `nil` back...